lib/generators/wcc/menu_generator.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Wcc
```

Comment:
this the correct namespace?

Reply:
Yes - if it's `WCC` then you have to `rails generate w_c_c:menu`

Reply:
inflections...